mockall/src/lib.rs

```diff
@@ -1231,6 +1233,17 @@ pub mod examples;
 /// let mock = MockBar::new();
 /// ```
 ///
+/// Mocking a trait variant with a different name requires adding a metaitem to the
+/// attribute in order to mock the variant instead of the original:
```

Comment:
Could you please clarify in the docs what automock will do in this instance?  Will it create a `MockFoo`, `MockLocalFoo`, or both?  I think the first one, but the docs should be explicit.